src/storm/modelchecker/prctl/helper/SparseMdpPrctlHelper.cpp

```diff
@@ -52,117 +54,125 @@ namespace storm {
+template<typename ValueType, typename SolutionType>
+std::map<storm::storage::sparse::state_type, SolutionType> SparseMdpPrctlHelper<ValueType, SolutionType>::computeRewardBoundedValues(
     Environment const& env, OptimizationDirection dir, rewardbounded::MultiDimensionalRewardUnfolding<ValueType, true>& rewardUnfolding,
     storm::storage::BitVector const& initialStates) {
-    storm::utility::Stopwatch swAll(true), swBuild, swCheck;
-
-    // Get lower and upper bounds for the solution.
-    auto lowerBound = rewardUnfolding.getLowerObjectiveBound();
-    auto upperBound = rewardUnfolding.getUpperObjectiveBound();
-
-    // Initialize epoch models
-    auto initEpoch = rewardUnfolding.getStartEpoch();
-    auto epochOrder = rewardUnfolding.getEpochComputationOrder(initEpoch);
-
-    // initialize data that will be needed for each epoch
-    std::vector<ValueType> x, b;
-    std::unique_ptr<storm::solver::MinMaxLinearEquationSolver<ValueType>> minMaxSolver;
-
-    ValueType precision = rewardUnfolding.getRequiredEpochModelPrecision(
-        initEpoch, storm::utility::convertNumber<ValueType>(storm::settings::getModule<storm::settings::modules::GeneralSettings>().getPrecision()));
-    Environment preciseEnv = env;
-    preciseEnv.solver().minMax().setPrecision(storm::utility::convertNumber<storm::RationalNumber>(precision));
-
-    // In case of cdf export we store the necessary data.
-    std::vector<std::vector<ValueType>> cdfData;
-
-    storm::utility::ProgressMeasurement progress("epochs");
-    progress.setMaxCount(epochOrder.size());
-    progress.startNewMeasurement(0);
-    uint64_t numCheckedEpochs = 0;
-    for (auto const& epoch : epochOrder) {
-        swBuild.start();
-        auto& epochModel = rewardUnfolding.setCurrentEpoch(epoch);
-        swBuild.stop();
-        swCheck.start();
-        rewardUnfolding.setSolutionForCurrentEpoch(epochModel.analyzeSingleObjective(preciseEnv, dir, x, b, minMaxSolver, lowerBound, upperBound));
-        swCheck.stop();
-        if (storm::settings::getModule<storm::settings::modules::IOSettings>().isExportCdfSet() &&
-            !rewardUnfolding.getEpochManager().hasBottomDimension(epoch)) {
-            std::vector<ValueType> cdfEntry;
-            for (uint64_t i = 0; i < rewardUnfolding.getEpochManager().getDimensionCount(); ++i) {
-                uint64_t offset = rewardUnfolding.getDimension(i).boundType == helper::rewardbounded::DimensionBoundType::LowerBound ? 1 : 0;
-                cdfEntry.push_back(storm::utility::convertNumber<ValueType>(rewardUnfolding.getEpochManager().getDimensionOfEpoch(epoch, i) + offset) *
-                                   rewardUnfolding.getDimension(i).scalingFactor);
+    if constexpr (std::is_same_v<ValueType, storm::Interval>) {
+        STORM_LOG_THROW(false, storm::exceptions::NotImplementedException, "We do not support computing reward bounded values with interval models.");
+    } else {
```

Comment:
Avoid line indentation for the remainder.
```suggestion
    STORM_LOG_THROW(constexpr (!std::is_same_v<ValueType, storm::Interval>), storm::exceptions::NotImplementedException, "We do not support computing reward bounded values with interval models.");
```